styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -252,6 +265,41 @@ private void run() {
     }
   }
 
+  private void workflowDelete() throws ExecutionException, InterruptedException {
+    final String component = namespace.getString(parser.workflowDeleteComponentId.getDest());
+    final String workflow = namespace.getString(parser.workflowDeleteWorkflowId.getDest());
+    final boolean force = namespace.getBoolean(parser.workflowDeleteForce.getDest());
+
+    if (System.console() != null && !force) {
+      final String reply = System.console().readLine(
+          "Sure you want to delete the workflow " + component + " " + workflow + " ? [yN]").trim();
+      if (!reply.equals("y")) {
+        System.exit(1);
+        return;
```

Comment:
unreachable code.

Reply:
iirc you have to do this to get rid of some warning, like intellij doesn't understand that `System.exit()` will not return